plugin/src/main/java/org/opensearch/ml/model/MLModelGroupManager.java

```diff
@@ -77,7 +77,7 @@ public void createModelGroup(MLRegisterModelGroupInput input, ActionListener<Str
                                     new IllegalArgumentException(
                                         "The name you provided is already being used by another model with ID: "
                                             + id
-                                            + ". Please provide a different name"
+                                            + ". Please provide a different name or add \"model_group_id\": \"lMPmr4kB4eSCtCCDmCDm\" to request body"
```

Comment:
why are we adding a hard coded model id in the error message?

Reply:
Oh, good catch, I need to clean up the code first.